materialize-motherduck/.snapshots/TestSQLGeneration

```diff
@@ -125,25 +125,6 @@ SELECT * FROM read_json(
 ) WHERE flow_document != '"delete"';
 --- End ""."a-schema".key_value storeQuery ---
 
---- Begin ""."a-schema".delta_updates loadQuery ---
```

Comment:
These two snapshots were bogus, since delta updates bindings never run load queries or merges. Note: A merge query in MotherDuck is accomplish by first deleting all the pre-existing rows that match the set of keys we are going to store, followed by inserting said rows to store.